2_process/src/create_site_list.R

```diff
@@ -140,3 +140,31 @@ transform_site_locations <- function(site_list_df,crs_out){
   
 }
 
+
+create_site_list_nontidal <- function(site_list_w_segs_df,mainstem_segs,fileout){
+  #' 
+  #' @description Function to filter sites thought to be influenced by tides 
+  #'
+  #' @param site_list_df data frame containing site locations and matched segment id's
+  #' @param mainstem_segs character string containing the mainstem segments that are assumed to be influenced by tides
+  #' @param fileout file path and name for output data, including the file extension
+  #' 
+  
+  site_list_filtered <- site_list_w_segs_df %>%
+    # For daily-resolution model, omit discrete sites associated with reaches thought to be tidal:
+    filter(!(subsegid %in% mainstem_segs & data_src_combined=="Harmonized_WQP_data")) %>%
```

Comment:
This filter removes discrete sites, but there are some discrete samples at NWIS sites that we do not want to include. The count of discrete sampling days is updated to be correct for those NWIS sites, but where are are the discrete samples removed somewhere?

Reply:
I'd prefer adding to this PR. Would it then be possible to update the `count_days_discrete` from the adjusted target instead of setting to 0 here? Or, maybe better, you could write a filtered site list from the sites in the adjusted target

Reply:
Yep, I could do that! I honestly kind of wavered back and forth between the two approaches so let's go with the version you suggest here.